src/tests/ftest/util/test_utils_pool.py

```diff
@@ -1006,8 +1026,10 @@ def pool_percentage_used(self):
         """
         daos_space = self.get_pool_daos_space()
         pool_percent = {
-            'scm': round(float(daos_space["s_free"][0]) / float(daos_space["s_total"][0]) * 100, 4),
-            'nvme': round(float(daos_space["s_free"][1]) / float(daos_space["s_total"][1]) * 100, 4)
+            'scm': round(float(daos_space["s_total"][0] - daos_space["s_free"][0]) /
+                         float(daos_space["s_total"][0]) * 100, 4),
```

Comment:
`W504 line break after binary operator`
We should result these. I think...
```suggestion
            'scm': round(float(daos_space["s_total"][0] - daos_space["s_free"][0])
                         / float(daos_space["s_total"][0]) * 100, 4),
```